python/Ganga/Core/GangaRepository/GangaRepositoryXML.py

```diff
@@ -936,6 +936,8 @@ def _actually_load_xml(self, fobj, fn, this_id, load_backup):
 
         if len(errs) > 0:
             logger.error("#%s Error(s) Loading File: %s" % (len(errs), fobj.name))
+            for err in errs:
+                logger.error("err: %s" % err)
```

Comment:
Without this change in loading the job from XML it's impossible to see why the job has been marked as corrupt. it's indexes removed and it's primary XML file destroyed as the backup was attempted to be loaded.